scripts/performance/configs/wan/__init__.py

```
@@ -0,0 +1,45 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    import megatron.bridge  # noqa: F401
+
+    HAVE_MEGATRON_BRIDGE = True
+except ModuleNotFoundError:
+    HAVE_MEGATRON_BRIDGE = False
+
+if HAVE_MEGATRON_BRIDGE:
+    from .wan_diffusion_pretrain import (
```

Flake8 (7.3.0)

[error] '.wan_diffusion_pretrain.wan_14b_pretrain_config_gb200' imported but unused

(F401)